phpBB/posting.php

```diff
@@ -1792,23 +1812,28 @@
 $notify_checked		= (isset($notify)) ? $notify : (($mode == 'post') ? $user->data['user_notify'] : $notify_set);
 
 // Page title & action URL
-$s_action = append_sid("{$phpbb_root_path}posting.$phpEx", "mode=$mode&amp;f=$forum_id");
-$s_action .= ($topic_id) ? "&amp;t=$topic_id" : '';
-$s_action .= ($post_id) ? "&amp;p=$post_id" : '';
+$s_action = append_sid("{$phpbb_root_path}posting.$phpEx", "mode=$mode");
 
 switch ($mode)
 {
 	case 'post':
+		$s_action .= ($forum_id) ? "&amp;f=$forum_id" : '';
```

Comment:
I am really not in favor of adding checks with these redundant parentheses.

Reply:
OK, it a personal style, I am happy to remove the parenthesis.

Reply:
Looking again, the parentheses are consistent the existing coding standard in this file, so i think that they should remain.

Reply:
Please remove those parenthesis though, those are just redundant and the existing codeing it is a bit old so it can not be taken as an example actually,  IMHO.

Reply:
Parenthesis are still there, let's get rid of them: devs preference > current code style > personal preference 😉.